### PR TITLE
DO NOT MERGE - Cbrune/bfd 368 api mappings - DO NOT MERGE

### PR DESCRIPTION
**Pull request by cbrunefearless**
### Change Details

Added the following fields to the api mappings.

CCLF Field Name | Origin |  
-- | -- | --
TAX_NUM | In Carrier Extract |  NOT MAPPED
BENE_DEATH_DT | In Beneficiary Extract |  
NCH_WKLY_PROC_DT | In Claims Extract |  
REV_CNTR_DT | Outpatient, HHA and Hospice Extracts |  
IME_OP_CLM_VAL_AMT | In Inpatient Extract |  
DSH_OP_CLM_VAL_AMT | In Inpatient Extract |  
CLM_HOSPC_START_DT_ID | In Hospice Extract |  
NCH_BENE_DSCHRG_DT | In Hospice Extract |  
CARR_CLM_CNTL_NUM | BCARRIER and DME Extract |  
FI_DOC_CLM_CNTL_NUM | Inpatient, Outpatient, HHA, Hospice, SNF |  
FI_ORIG_CLM_CNTL_NUM | Inpatient, Outpatient, HHA, Hospice, SNF |  
CLM_UNCOMPD_CARE_PMT_AMT | In Inpatient Extract |  
EFCTV_BGN_DT | In Beneficiary Extract |  
EFCTV_END_DT | In Beneficiary Extract |  
BENE_LINK_KEY | In Beneficiary Extract | NOT MAPPED
CLM_DISP_CD |   |  

This pr is for new fields that were added to the API.
### Feedback Requested

Map sure the mappings make sense and they are in the correct spot.
### External References

- [BFD-368-Api mappings for new fields](https://jira.cms.gov/browse/BFD-368)

### Security Implications

- [ ] new software dependencies
NO

- [ ] altered security controls
NO

- [X ] new data stored or transmitted

- [ ] security checklist is completed for this change
No

- [ ] requires more information or team discussion to evaluate security implications

### CODE REVIEW

**Comment by jzulim** [comment]
Great work on this! A few notes:
- What was CCW's 'stance' on why some of these fields are missing from their codebook? If they started landing in the codebook at some point, would we break?
- We'll need to map the new fields to the R4PatientResourceProvider. 

Will cirle back after the R4 additions. Thx!

File: apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/HospiceClaimTransformer.java [hunk 1]
```diff
@@ -86,7 +88,6 @@ private static ExplanationOfBenefit transformClaim(HospiceClaim claimGroup) {
         Optional.of(claimGroup.getUtilizationDayCount()));
 
     if (claimGroup.getHospicePeriodCount().isPresent()) {
-      // TODO should this be benefitBalance?
```
Comment (jarwilliams1971) [inline apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/HospiceClaimTransformer.java:89]: From what I can see that field should not be mapped as the benefitBalance. Benefit Balance is used  for the actual Balance by Benefit Category overall. Not what was used per claim.

**Review by jarwilliams1971 — APPROVED** [review]

**Review by jzulim — DISMISSED** [review]
Logic looks good, patterns, naming, comments, etc. Mapping has been reviewed by smarter people. LGTM.

**Review by jzulim — APPROVED** [review]
Lgtm

**Review by jarwilliams1971 — APPROVED** [review]

**Pull request merged**